jsonnet/kube-prometheus/jsonnetfile.json

```diff
@@ -35,7 +35,7 @@
           "subdir": "jsonnet/mixin"
         }
       },
-      "version": "master"
+      "version": "release-0.44"
```

Comment:
Sorry for not pointing out, this can also stay at master and can just be done as a PR towards the release branch, otherwise we need to change it twice as in master it will always stay master. If that makes sense?

Reply:
Makes sense, sorry I misunderstood your first comment about this. It should be good now.